scipy/stats/_stats_py.py

```diff
@@ -572,75 +568,19 @@ def mode(a, axis=0, nan_policy='propagate', keepdims=None):
     To get mode of whole array, specify ``axis=None``:
 
     >>> stats.mode(a, axis=None, keepdims=True)
-    ModeResult(mode=[3], count=[5])
+    ModeResult(mode=[[3]], count=[[5]])
```

Comment:
This is the problem report in gh-17214. I think I let it slip when I was adding `keepdims` because this had been the behavior of `mode` before `keepdims` was introduced.